spec/components/previews/layouts/component_preview.html.erb

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
Took head from main layout, probably don't need everything in here

Reply:
Good stuff, yeah I think virtually all of this can go, we might just need the js and style links.